src/SincCoeffs.cc

```diff
@@ -39,7 +39,13 @@ namespace base {
 namespace {
 
 // Convenient wrapper for a Bessel function
-inline double J1(double const x) { return boost::math::cyl_bessel_j(1, x); }
+inline double J1(double const x) {
```

Comment:
`const` is redundant if passed by value

Reply:
I'm not going to get into changing things outside the scope of the fix at this time.